api/controllers/swagger.go

```diff
@@ -30,6 +36,11 @@ func (s *SwaggerController) Routes() []*fireball.Route {
 	return routes
 }
 
+func (s *SwaggerController) Root(c *fireball.Context) (fireball.Response, error){
+	html := "<a href='/api/?url=/swagger.json'>Swagger</a>"
+	return fireball.NewResponse(200, []byte(html), nil), nil
```

Comment:
Nice add. I like it.

Reply:
Rather than rendering html with a link to the base swagger definitions, could we do some sort of URL rewriting or permanent redirection if the user hits the root of the API?

Reply:
I'm using this as a healthcheck, so i want to return a 200, not a 300